appsec/appsec.go

```diff
@@ -138,18 +138,11 @@ func TrackCustomEvent(ctx context.Context, name string, md map[string]string) {
 
 // Return the root span from the span stored in the given Go context if it
 // implements the Root method. It returns nil otherwise.
```

Comment:
That comment seems outdated (apparently, `tracer.Span` declares the `Root` method.